03_agent_deployment.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+set -xe
```

Comment:
let's also set "unset variables as an error" and "exit on any error in pipeline"